README.md

```diff
@@ -14,6 +14,12 @@ This repository contains a Rust wrapper around the [PSA Cryptography API](https:
 
 * [`psa-crypto`](psa-crypto): a higher-level, more Rust-friendly interface
 
+In order to cross-build for ARM using the "ci.sh" script, the gcc for arm needs to be installed:
+
+    sudo apt install gcc-12-arm-linux-gnueabihf gcc-arm-linux-gnueabihf \
```

Comment:
It seems `sudo apt install gcc-arm-linux-gnueabihf gcc-aarch64-linux-gnu` would suffice, and this works in more general situations.

(We can automatically get `gcc-VERSION-arm-linux-gnueabihf` and `gcc-VERSION-aarch64-linux-gnu` as dependencies, where "VERSION" is a gcc's version number depending on the Linux distributions.)